articles/app-service/includes/quickstart-php/quickstart-php-azure-portal-pivot.md

```diff
@@ -0,0 +1,200 @@
+
+## Fork the sample repository
+
+1. In your browser, navigate to the repository containing [the sample code](https://github.com/Azure-Samples/php-docs-hello-world).
+
+2. In the upper right corner, select **Fork**.
+
+    ![In GitHub, in the Azure-Samples/php-docs-hello-world repo, select Fork.](../../media/quickstart-php/fork-php-docs-hello-world-repo.png)
+
+3. On the **Create a new fork** screen, confirm the **Owner** and **Repository name** fields. Select **Create fork**.
+
+    ![In GitHub, in the Azure-Samples/php-docs-hello-world repo, on the Create a new fork screen, confirm the details for the Owner and Repository name fields.](../../media/quickstart-php/fork-details-php-docs-hello-world-repo.png)
+
+    >[!NOTE]
+    > This should take you to the new fork. Your fork URL will look something like this: https://github.com/YOUR_GITHUB_ACCOUNT_NAME/php-docs-hello-world
+
+### Update the fork's default branch
+
+1. From the fork repository, select **Settings**.
+
+ ![In GitHub, in the forked php-docs-hello-world repo, select Settings.](../../media/quickstart-php/forked-php-docs-hello-world-repo-settings.png)
+
+2. Under **Code and automation**, select **Branches**.
+
+ ![In GitHub, in the forked php-docs-hello-world repo's Settings, select Branches.](../../media/quickstart-php/forked-php-docs-hello-world-repo-settings-branches.png)
+
+3. Under **Default branch**, to the right of `master`, select the **Rename branch** button represented by a pencil icon.
+
+ ![On the Branches for the forked java-docs-spring-hello-world repo, select the Rename branch button after master.](../../media/quickstart-php/forked-php-docs-hello-world-repo-settings-branches-edit.png)
+
+4. In the **Rename this branch** pop-up, enter `main`.
+
+ ![In the Rename this branch dialog, enter 'main'.](../../media/quickstart-php/forked-php-docs-hello-world-repo-rename-this-branch.png)
+
+5. Select **Rename branch**.
+
+> [!TIP]
+> The branch name change isn't required by App Service. However, since many repositories are changing their default branch to `main`, this quickstart also shows you how to deploy a repository from `main`.
+
+## Deploy to Azure
+
+### Sign in to Azure portal
+
+Sign in to the Azure portal at https://portal.azure.com.
+
+
+### Create Azure resources
+
+1. At the top of the portal, type **app services** in the search box. Under **Services**, select **App Services**.
+
+ ![In the Azure portal, search for app services. Then select App Services.](../../media/quickstart-php/azure-portal-search-for-app-services.png)
+
+2. On the **App Services** page, select **Create**.
+
+ ![In the Azure portal, on the App Services page, select Create from the actions bar.](../../media/quickstart-php/azure-portal-create-app-service.png)
+
+3. In the **Basics** tab, under **Project details**, ensure the correct subscription is selected and then select to **Create new** resource group. Type *myResourceGroup* for the name. Then, select **OK**.
```

Comment:
For steps 3-5, consider using just 1 step with a table for each setting, and one image to cover. as convention, we use a 3pt red square to highlight all the UI elements touched, no need to number them in the screenshot.

again, for the image I would just reduce the whitespace on the right so that everything else can be bigger in the screenshot. I would also say the little pop-up from "Create new" is not worth showing since it's pretty self-explanatory.